src/DotNet/Writer/MethodBodyWriter.cs

```diff
@@ -84,6 +107,11 @@ public MethodBodyWriter(ITokenProvider helper, CilBody cilBody, bool keepMaxStac
 			this.helper = helper;
 		}
 
+		internal void Reset(MethodDef method, bool keepMaxStack) {
+			this.method = method;
+			Reset(method.Body, keepMaxStack);
+		}
+
 		internal void Reset(CilBody cilBody, bool keepMaxStack) {
```

Comment:
Is this method even called other than from the new `Reset()` method? If it only has one caller, it can be inlined in it.